packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayerCallbacks.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.videoplayer;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+/**
+ * Callbacks representing events invoked by {@link VideoPlayer}.
+ *
+ * <p>In the actual plugin, this will always be {@link VideoPlayerEventCallbacks}, which creates the
+ * expected events to send back through the plugin channel. In tests methods can be overridden in
+ * order to assert results.
+ */
+interface VideoPlayerCallbacks {
```

Comment:
Maybe link the `Player.Listener` [docs](https://developer.android.com/reference/androidx/media3/common/Player.Listener) to the callbacks that directly map to one of those events?

Reply:
Done!